### PR TITLE
Closes #825: Untangle Match Group services 

### LINKED ISSUE #825: Check Match Group's services
As discussed in #748: There are many services run by the [Match Group](https://github.com/datenanfragen/data/blob/master/companies/match-group.json) (thanks @rugk :D). We want to create new records for those that have their own privacy-related email address and bundle these that share an email.

- [x] "Plenty Of Fish (pof.com)",
- [x] "OkCupid (okcupid.com)",
- [x] "Meetic (meetic.fr)",
- [x] "Matchaffinity",
- [x] "LoveScout24",
- [x] "Twoo (twoo.com)",
- [x] "Neu.de",
- [x] "Partner.de",
- [x] "Amoureux (amoureux.com)",
- [x] "Lexamore.nl",
- [x] "DisonsDemain.fr",
- [x] "BlackPeopleMeet.com",
- [x] "OurTime",
- [x] "Pairs",
- [x] "ParPerfeito",
- [x] "LDSPlanet.com",
- [x] "PetPeopleMeet.com",
- [x] "BBPeople (bbpeoplemeet.com)",
- [x] "InterracialPeopleMeet.com",
- [x] "Chemistry.com",
- [x] "MarriageMindedPeopleMeet.com",
- [x] "LatinoPeopleMeet.com",
- [x] "DemocraticPeopleMeet.com",
- [x] "RepublicanPeopleMeet.com",
- [x] "BabyBoomerPeopleMeet.com",
- [x] "singleparentmeet.com",

### PR DESCRIPTION
**Pull request by baltpeter**
Supersedes #1085.

Wow, what a mess. :D

### CODE REVIEW

File: companies/matchaffinity-dk.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    "relevant-countries": [
+        "dk"
+    ],
+    "categories": [
+        "social media"
+    ],
+    "name": "MatchAffinity.dk",
+    "address": "Match.com Nordic AB\nc/o Deloitte\nBox 386\n201 23 Malmö\nSweden",
+    "email": "privacy@matchaffinity.dk",
+    "web": "https://www.matchaffinity.dk/pages/misc/privacy",
```
Comment (mal-tee) [inline companies/matchaffinity-dk.json:12]: I think `web` and `sources` should be swapped :D

File: companies/matchaffinity-no.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    "relevant-countries": [
+        "no"
+    ],
+    "categories": [
+        "social media"
+    ],
+    "name": "MatchAffinity.no",
+    "address": "Match.com Nordic AB\nc/o Deloitte\nBox 386\n201 23 Malmö\nSweden",
+    "email": "privacy@matchaffinity.no",
+    "web": "https://www.matchaffinity.no/pages/misc/privacy",
```
Comment (mal-tee) [inline companies/matchaffinity-no.json:12]: web <-> sources

File: companies/matchaffinity-se.json [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    "relevant-countries": [
+        "se"
+    ],
+    "categories": [
+        "social media"
+    ],
+    "name": "MatchAffinity.se",
+    "address": "Match.com Nordic AB\nc/o Deloitte\nBox 386\n201 23 Malmö\nSweden",
+    "email": "privacy@matchaffinity.se",
+    "web": "https://www.matchaffinity.se/pages/misc/privacy",
```
Comment (mal-tee) [inline companies/matchaffinity-se.json:12]: web <-> sources

File: companies/ourtime-se.json [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+    "categories": [
+        "social media"
+    ],
+    "name": "Match.com Nordic AB",
+    "runs": [
+        "se.ourtime.com"
+    ],
+    "address": "c/o Deloitte\nBox 386\n201 23 Malmö\nSweden",
+    "email": "privacy-se@europe.ourtime.com",
+    "web": "https://se.ourtime.com/pages/misc/privacy",
```
Comment (mal-tee) [inline companies/ourtime-se.json:15]: web <-> sources

File: companies/partner-de.json [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+    "name": "partner.de",
+    "runs": [
+        "partner.ch"
+    ],
+    "address": "c/o NEU.DE GmbH\nNeuturmstraße 5\n80331 München\nDeutschland",
+    "email": "privacy@partner.de",
+    "web": "https://www.partner.de/",
+    "sources": [
+        "https://www.partner.de/pages/misc/privacy",
+        "https://www.partner.de/pages/misc/impressum",
```
Comment (mal-tee) [inline companies/partner-de.json:16]: > Kontaktieren Sie bitte Herrn [REDACTED] per E-Mail unter datenschutz[at]neu.de.

I agree, privacy@partner.de seems more fitting

**Review by mal-tee — CHANGES_REQUESTED** [review]
Wow, thats a lot! :o Thank you very much. I've left some small notes.

File: companies/match-gb.json [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+    "categories": [
+        "social media"
+    ],
+    "name": "Match.com",
+    "runs": [
+        "uk.match.com",
+        "ie.match.com"
+    ],
+    "address": "MIL\nc/o Skadden\n40 Bank Street\nCanary Wharf\nLondon\nE14 5DS\nUnited Kingdom",
+    "email": "privacy@match.co.uk",
```
[the remark this answers is not in the document]
Comment (baltpeter) [reply companies/match-gb.json:16]: Probably the same thing, considering match.co.uk redirects to uk.match.com.

**Review by mal-tee — APPROVED** [review]
Thank you!

**Pull request merged**